src/app/components/mno-app-features/mno-app-features.coffee

```diff
@@ -0,0 +1,15 @@
+@App.directive('mnoAppFeatures', ->
+  return {
+    restrict: 'EA'
+    templateUrl: 'app/components/mno-app-features/mno-app-features.html',
+    scope: {
+      app: '='
+    }
+
+    controller: ($scope) ->
+      $scope.isSingleBilling = !!$scope.app.single_billing
+      $scope.isDataSharing = !!$scope.app.is_connec_ready
+      $scope.isSingleSignOn = !!$scope.app.add_on
```

Comment:
An app can have SSO, and not be an add-on (e.g: Appointed)

Reply:
@manu-d Is there another indicator for sso ?

Reply:
It looks like the apps are being fetched using the `/marketplace` endpoint. You can add a line in this file (https://github.com/maestrano/mno-enterprise/blob/4.0/api/app/views/mno_enterprise/jpi/v1/marketplace/_app.json.jbuilder#L12) so that there will be an attribute for `$scope.app.sso_enabled` 

It could just be `json.sso_enabled app.sso_enabled?` since the apps already have a method `#sso_enabled?`

Reply:
Great, thanks @adamaziz15